container.go

```diff
@@ -456,6 +456,20 @@ func (c *Container) start() error {
 	}
 	defer c.pod.proxy.disconnect()
 
+	// Try to check for block storage if Fstype is empty.
+	// If Fstype has been already assigned, then block device has been checked and assigned earlier.
+	// Worst case, we perform the check twice for non-block storage.
+	if c.state.Fstype == "" {
```

Comment:
I get that you use this `string` as a flag, but it might be better to try to improve this for more readability. Maybe introducing a `bool` in order to indicate this container has already been checked/processed.